srcpkgs/wren-cli/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'wren-cli'
+pkgname=wren-cli
+version=0.3.0
+revision=1
+build_wrksrc=projects/make
+build_style=gnu-makefile
+make_build_args="config=release_64bit"
+short_desc="Small, class-based, concurrent scripting language - CLI"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="MIT"
+homepage="http://wren.io/cli"
+distfiles="https://github.com/wren-lang/wren-cli/archive/${version}.tar.gz"
+checksum=a498d2ccb9a723e7163b4530efbaec389cc13e6baaf935e16cbd052a739b7265
+
+pre_fetch() {
+	if [ ${XBPS_TARGET_WORDSIZE} -eq 32 ]; then
+		make_build_args="config=release_32bit"
+	fi
+}
+
+do_install() {
+	vbin bin/wren_cli wren
```

Comment:
Yeah, this path needs to be relative to the build_wrksrc, so something like `../../bin/wren_cli`. At most you could do `${wrksrc}/bin/wren_cli`, but I don't remember if wrksrc is an absolute path, which would be required for this to work.